### PR TITLE
ensure there is always content on the landing page

### PR DESCRIPTION
**Pull request by Pomax**
content subject to copy review and styling.

fixes https://github.com/mozilla/mozillafestival.org/issues/565

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -33,7 +33,50 @@
   </script>
 </head>
 <body>
-  <div id="my-app"></div>
+  <div id="my-app">
+
```
Comment (mmmavis) [inline public/index.html]: @Pomax can we have note in here saying this is for "no js is allowed" scenario?

Comment (Pomax) [inline public/index.html]: of course, feel free to edit the PR file with an HTML comment

File: public/index.html [hunk 2]
```diff
@@ -33,7 +33,50 @@
...
+      </div>
+      <div class="jumbotron-container">
+        <div class="jumbotron m-0" style="background-image: url(&quot;/assets/images/hero/home/banner-home_5.jpg&quot;);">
+          <div class="content-wrapper">
+            <h1>MozFest</h1>
+            <h2>The world's leading festival for the open Internet movement.</h2>
+            <div class="horizontal-rule"></div>
+            <p>October 27-29, 2017 Ravensbourne College, London</p>
+            <a href="https://vimeo.com/205552025/37560e3619" class="btn p-3 m-3">Watch Video</a>
+            <div class="content-strong">
```
Comment (Pomax) [inline public/index.html]: I kind of guessed at this class, I don't know if this exists or not.

**Comment by mmmavis** [comment]
Got some design help from @sabrinang . Merging this PR now.

**Comment by Pomax** [comment]
🎉

**Pull request merged**